dlog/commitment/src/srs.rs

```diff
@@ -22,6 +22,42 @@ pub struct SRS<G: CommitmentCurve> {
     pub endo_q: G::BaseField,
 }
 
+/// A trimmed reference string. It contains a reference to an SRS, along with a
+/// trimmed power-of-2 length.
+pub struct TrimmedSRS<'a, G: CommitmentCurve> {
+    srs: &'a SRS<G>,
```

Comment:
This is accessed in `commitment.rs`, it should probably be public.

Reply:
(getter here is better as you don't want to let someone set that manually.)